Docs: Write MVP for Trino docs (#952)
hasura/ddn-docs: docs/how-to-build-with-ddn/with-trino.mdx

      echo 'connection-user=myuser' >> /etc/trino/catalog/postgres.properties &&
      echo 'connection-password=mypassword' >> /etc/trino/catalog/postgres.properties &&
      /usr/lib/trino/bin/run-trino
      "
    healthcheck:
      test: ["CMD", "curl", "-f", "http://localhost:8080/v1/info"]
      interval: 10s
      timeout: 5s
      retries: 5

volumes:
  pgdata:
```

```sh title="Run the container:"
docker compose -f app/connector/my_trino/compose.trino.yaml up -d
```

The Trino container will start; you can access the UI — which will give you general information about your clusters —
here: [`http://localhost:8080/`](http://localhost:8080/).

### Step 5. Create a table in your PostgreSQL database

```sh title="Enter the PostgreSQL container:"
docker exec -it postgres psql -U myuser -d mydb
```

```sql title="Then, create your first table and seed the database:"
--- Create the table
CREATE TABLE users (
  id SERIAL PRIMARY KEY,
  name TEXT NOT NULL,
  age INT NOT NULL
);

--- Insert some data
INSERT INTO users (name, age) VALUES ('Alice', 25);
INSERT INTO users (name, age) VALUES ('Bob', 30);
INSERT INTO users (name, age) VALUES ('Charlie', 35);
```

> You can verify this worked by using Adminer to query all records from the `users` table:

Reviewer: ```suggestion
You can verify this worked by querying the users table directly in the psql session:
```

```sql
SELECT * FROM users;
```

### Step 6. Introspect your Trino instance

```sh title="Next, use the CLI to introspect your Trino database:"
ddn connector introspect my_trino
```

After running this, you should see a representation of your database's schema in the
`app/connector/my_trino/configuration.json` file; you can view this using `cat` or open the file in your editor.

```sh title="Additionally, you can check which resources are available — and their status — at any point using the CLI:"
ddn connector show-resources my_trino
```

### Step 7. Add your model

```sh title="Now, track the table from Trino server as a model in your DDN metadata:"

> ddn model add my_trino "users"

Reviewer: Other tuts we're not quoting these. Does it make a difference.

```suggestion
ddn model add my_trino users
```

Author: Good point! During one of the guides, I was using `"*"` for all models and I think I just got back in the habit.

  (1, 'My First Post', 'This is Alice''s first post.'),
  (1, 'Another Post', 'Alice writes again!'),
  (2, 'Bob''s Post', 'Bob shares his thoughts.'),
  (3, 'Hello World', 'Charlie joins the conversation.');
```

```sql title="To verify this, query the joined data:"
SELECT
  posts.id AS post_id,
  posts.title,
  posts.content,
  posts.created_at,
  users.name AS author
FROM
  posts
JOIN
  users ON posts.user_id = users.id;
```

You should see a list of posts returned with the author's information joined from the `users` table

### Step 12. Refresh your metadata and rebuild your project

:::tip

The following steps are necessary each time you make changes to your **source** schema. This includes, adding,
modifying, or dropping tables.

:::

#### Step 12.1. Re-introspect your data source

```sh title="Run the introspection command again:"
ddn connector introspect my_trino
```

In `app/connector/my_trino/configuration.json`, you'll see schema updated to include operations for the `posts` table.
In `app/metadata/my_trino.hml`, you'll see `DOCS.PUBLIC.POSTS` present in the metadata as well.

#### Step 12.2. Update your metadata

```sh title="Add the posts model:"

> ddn model add my_trino "posts"

Reviewer: ```suggestion
ddn model add my_trino posts
```

Just for consistency with other tuts?
